packages/nuxt/src/app/composables/asyncData.ts

```diff
@@ -333,7 +339,7 @@ export function useAsyncData<
     return nuxtApp._asyncDataPromises[key]!
   }
 
-  asyncData.clear = () => clearNuxtDataByKey(nuxtApp, key)
+  asyncData.reset = asyncData.clear = () => clearNuxtDataByKey(nuxtApp, key)
```

Comment:
Should we also rename `clearNuxtDataByKey` to `resetNuxtDataByKey`?

Reply:
Maybe part of a bigger question - should we rename `clearNuxtData`? In fact I am wondering if it would be better to keep `clear`/`clearNuxtData` as `reset` is a change and in most cases the default value _is_ the empty state of the composable.